### ISSUE TITLE
AA Playlists Non-Functioning and Disability All Volume Controls

### ISSUE THREAD

**Issue by DevilsDesigns** [jellyfin/jellyfin-android#387]
**Describe the bug**
Android Auto and the Official Jellyfin app will either not play playlists longer than about 20 titles if ever. As well as if you try to start it with your phone the music will play but all volume up and down and mute buttons on the head unit with Wireless android auto will not function until you either turn off the car and back on or disconnect device from the head unit. (Since im using AA wireless all i do is disconnect Wifi and Bluetooth then turn on wifi again which it will auto reconnect to my head unit then turn on bluetooth and reconnect to head unit as well). I saw something about this on the Jellyfin forums upon the first release of AA support for Jellyfin. I have seen no further development on the matter.

**System (please complete the following information):**
 - OS: Windows (Hosting)
 - Clients: Official Android Jellyfin App
 - Jellyfin Version: 10.7.5
 - Playback: Transcoding on Nvidia Geoforce GTX 960 through Nvidia NVENC
 - Reverse Proxy: Caddy v2
 - Base URL: tv.movies4you.digital
 - Networking: Wired Ethernet
 - Storage: local
- Head Unit with Wireless AA: Pioneer AVH-W4500NEX
- Steering Wheel Controls: Factory Retained 2011 Subaru Forester 
**To Reproduce**

1. Install Official Jellyfin app from F-Droid
2. Make a Playlist on PC with more than 100 Songs.
3. Save the playlist
4.  Load up Jellyfin on Wireless AA.
5. Go to playlists
6. Select Playlist-Name-HERE
7. It wont let you select the playlist. You can click it 100 times it will show the click but nothing will happen.
8. So load up Jellyfin app on your phone.
9. Click Playlists and select the Playlist-Name-HERE you were trying to select on Wireless AA.
10. Starts to play
11. You cannot now adjust sound or mute. The functions for steering wheel controlls but also physical head unit will not allow the buttons to work. 
12. 

**Expected behavior**
Jellyfin should be able to have no limit to the max amount of songs in a playlist through Wireless AA headunit and continue to have working volume and mute controls. Also when loading up Jellyfin app on phone you should still retain functionability on other devices when streaming music if you want to manually select a playlist on your Jellyfin App on your android phone to play on AA when connected or Bluetooth off a aftermarket head unit.

**Logs**
none

**Screenshots**
I can upload a short video if needed please let me know.

**Comment by DevilsDesigns** [comment]
Is there anyone that can help?

**Comment by Maxr1998** [comment]
When using the app on the phone, another player is used, which doesn't support the remote volume controls, unfortunately. The playlist length issue is a known bug that we still have to fix.

**Comment by Maxr1998** [comment]
Please confirm whether your issue persists. Thank you.

**Comment by wchesley** [comment]
Sorry to necro this issue, however I have noticed that the issue persists when playing playlists over 100 songs, the application only see's the first 100 songs and nothing past that.

**Comment by DevilsDesigns** [comment]
> Sorry to necro this issue, however I have noticed that the issue persists when playing playlists over 100 songs, the application only see's the first 100 songs and nothing past that. 

This is just how jellyfin works. It won't load more than 100 songs inside playlist or outside. It apparently may be fixed in 10.9. but no promises

**Comment by GanimanSwift** [comment]
I am not sure #1903 is a duplicate of this exactly. Either way, this issue makes the Android Auto functionality of this app not a great user experience. Accessing media outside of this limitation requires going right to the phone (while driving, in theory) to select media, which then also does not appear on the Android Auto display. You can hear it, but you can't see it nor control it via Android Auto controls. Its just dangerous and deserves attention. One of my suggestions is to use pages, if that works. Somehow this should be addressed. What will it take? Does this effort need to be sponsored?

**Comment by DevilsDesigns** [comment]
Agreed this is from my post 5 years ago and no update unfortunately. > I am not sure #1903 is a duplicate of this exactly. Either way, this issue makes the Android Auto functionality of this app not a great user experience. Accessing media outside of this limitation requires going right to the phone (while driving, in theory) to select media, which then also does not appear on the Android Auto display. You can hear it, but you can't see it nor control it via Android Auto controls. Its just dangerous and deserves attention. One of my suggestions is to use pages, if that works. Somehow this should be addressed. What will it take? Does this effort need to be sponsored?

**Comment by nielsvanvelzen** [comment]
Anything related to the lack of pagination in our Android Auto implementation is effectively the same issue. I've been working on a new implementation (for other reasons) and one of the things I'm stuck on is the pagination, as Android Auto just plainly does not support it. Google even proudly says so in their documentation. This applies to everything like albums, artists, tracks, playlists etc.
